lib/linter/rules.js

```diff
@@ -56,7 +56,7 @@ class Rules {
      */
     get(ruleId) {
         if (typeof this._rules[ruleId] === "string") {
-            this.define(ruleId, require(this._rules[ruleId]));
+            this.define(ruleId, require(`../rules/${this._rules[ruleId]}`));
```

Comment:
Can you explain why changing it? a rule may be loaded from a third-party plugin, so this won't always work.

Reply:
I see. Transpiling dynamic CJS imports into ESM is kinda of tricky so this was for now the only way to workaround it. I was able to verify that this code is not touched during the tests so it shouldn't be modified. I will work on reverting it.